src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
The problem is that the newer versions of h2o-genmodel are not compatible with your test model, so we get failed tests when I build with an updated pom.xml.

````
...nested exception is java.lang.NoSuchMethodError: hex.genmodel.GenModel.getPrediction([D[DD)I] with root cause

java.lang.NoSuchMethodError: hex.genmodel.GenModel.getPrediction([D[DD)I
	at gbm_afae103d_92ee_4059_93e5_eac979857a55.score0(gbm_afae103d_92ee_4059_93e5_eac979857a55.java:108)
	at org.trustedanalytics.h2oscoringengine.h2omodel.H2oModel.score(H2oModel.java:76)
	at org.trustedanalytics.h2oscoringengine.rest.H2oArrayScoringEngineController.score(H2oArrayScoringEngineController.java:54)
....
````

I might be able to fix this if you can provide the source for the test model jars in src/test/resources. Or maybe you have another suggestion?